src/elements/content-sidebar/additional-tabs/AdditionalTabFtuxTooltip.scss

```diff
@@ -1,3 +1,3 @@
-.rec-app-ftux-tooltip.tooltip {
+.bdl-AdditionalTabFtuxTooltip.tooltip {
```

Comment:
Is the `.tooltip` part of the selector needed?

Reply:
Without it, `.bdl-GuideTooltip.bdl-Tooltip` wins out and overrides the max-width